src/pages/ShoppingCart/utils/calculateTotalPrice.ts

```diff
@@ -0,0 +1,13 @@
+import { ShoppingCartProduct } from './concatProductsAndQuantity';
+
+export const calculateTotalPrice = (
+  cartProducts?: ShoppingCartProduct[]
+): string => {
+  const prices = cartProducts?.map(
+    (cartProduct) => cartProduct.product.base_price_sol * cartProduct.quantity
+  );
+
+  const total = prices?.reduce((a, b) => a + b).toFixed(2);
```

Comment:
What about longer names for clarity? For example
```suggestion
  const total = prices?.reduce((current, sum) => current + sum).toFixed(2);
```

maybe @anaceciliamolinares can help us with good names

Reply:
Thank u !, fixed